model/DataBaseAccess.php

```diff
@@ -177,7 +178,28 @@ public function addMultiplePermissions(array $permissionData)
                 }
             }
         }
-        $this->getPersistence()->insertMultiple(self::TABLE_PRIVILEGES_NAME, $insert);
+
+        $this->getLogger()->debug(
+            "Processing {count} permission inserts in {chunks} chunks",
+            [
+                'count' => count($insert),
+                'chunks' => ceil(count($insert)/self::INSERT_CHUNK_SIZE)
+            ]
+        );
+
+        foreach(array_chunk($insert, self::INSERT_CHUNK_SIZE) as $index => $batch) {
```

Comment:
```suggestion
        foreach (array_chunk($insert, self::INSERT_CHUNK_SIZE) as $index => $batch) {
```

Reply:
Already fixed in 0c6e230eaa4b7ffc1a0225ae00c07c1dd8942614 (I should have run the linter before pushing the branch :sweat_smile: )